### ISSUE TITLE
[feature request] Custom playlists

### ISSUE THREAD

**Issue by payasrelekar** [flackbash/AudioAnchor#108]
I have few language learning audio books, spread over multiple courses. e.g. German has Foundation 1-2-3-4, Builder 1-2-3-4, Advanced 1-2-3-4, Vocab 1-2-3-4 and so on. Same goes for other languages.

As such the directory structure is of multiple layers, and alphabetically Advanced always comes first. Currently AudioAnchor, as far as I know, only picks the directory of the actual audio file as the book and allows going through it at once as single playlist. I wouls like to extend this, and build, for ex. single 'German Foundation 1-2-3-4 -> Builder 1-2-3-4 -> Advanced 1-2-3-4 -> ...' playlist, and keep track of my progress throughout the entire course.

It would be nice if the progress is maintained over entire playlist, but that is optional. A simple way to maintain and be able to see bird's eye view and resume from last played position over the complete list would be sufficient.

Overall fantastic, lightweight, pretty and extremely useful app! Thanks a lot!

**Comment by flackbash** [comment]
I agree that adding playlist support could be useful. However, this will require quite some changes in the structure of the app and is is not an improvement that I will focus on any time soon.

Right now, to have a "playlist" 'German Foundation 1-2-3-4', all audio files belonging to this playlist would have to be in the same directory.

Glad you like the app :)
